client/components/main/layouts.jade

```diff
@@ -13,6 +13,13 @@ template(name="userFormsLayout")
     h1.at-form-landing-logo
       img(src="{{pathFor '/wekan-logo.png'}}" alt="Wekan")
     +Template.dynamic(template=content)
+    div.at-form-lang
+      select.select-lang.js-userform-set-language
```

Comment:
Actually the reason why I would like the user to explicitly check the language before registration is because I plan to revive the “Welcome Board” feature. Basically it’s a board filled with some data (like Wekan explanations) so that that the first page that the user see after registration is not a completely empty one and he can start playing with some data.

This board will need to be created (or rather copied from a template) just after the user has registered, and it will not be possible to change the language of the content (card titles, descriptions, comments) after its instantiation.

And so yes, if we go on that path we should also add this field in the enroll by email form.